Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
+FROM alpine
```

Comment:
flag the version, we don't want another update of alpine which will break everything